test/airdrop-test.js

```diff
@@ -14,7 +14,8 @@ const AirdropProof = require('../lib/primitives/airdropproof');
 const network = Network.get('regtest');
 
 const workers = new WorkerPool({
-  enabled: false
+  enabled: false,
+  size: 2
```

Comment:
still need size when disabled?

Reply:
No, it's not necessary. But I still added in case we change those tests in the future.